layout-management-services/src/main/java/org/exoplatform/layoutmanagement/rest/SiteNavigationRestService.java

```diff
@@ -106,7 +112,11 @@ public Response createNode(@Parameter(description = "navigation node id")
         return Response.status(Response.Status.UNAUTHORIZED).build();
       }
       NodeState nodeState;
-      nodeState = new NodeState(nodeLabel, null, -1, -1, isVisible ? Visibility.DISPLAYED : Visibility.HIDDEN, null, null);
+      if (startPostDate != null && endPostDate != null) {
```

Comment:
- I think that we should also:
  - Pass the schedule boolean value and test on it
  - Test on isVisible true value